metrics/ethereum.py

```
@@ -255,23 +255,23 @@ def process_data(self, block) -> float:
-    TOKEN_CONTRACTS: dict[str, str] = {
-        "ethereum": "0xdAC17F958D2ee523a2206206994597C13D831ec7",  # USDT
-        "base": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",  # USDC
-        "arbitrum": "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9",  # USDT
-        "bnb": "0x55d398326f99059fF775485246999027B3197955",  # BSC-USD
+    # WETH Deposit events - happen regularly but not too frequently
+    WETH_CONTRACTS: dict[str, str] = {
+        "ethereum": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",  # WETH
+        "base": "0x4200000000000000000000000000000000000006",  # WETH on Base
+        "arbitrum": "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",  # WETH on Arbitrum
+        "bnb": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",  # WBNB
     }
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)